starwars/src/components/StarwarsCharacter.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "../App.css";
+
+const StarwarsCharacter = props => {
+  return (
+    <div className="characteristics">
+      <figure className="title">
```

Comment:
```suggestion
      <div className="title">
```
Don't forget your HTML basics, a figure is a semantic tag for an image or an image with a caption